rust/agents/relayer/src/msg/serial_submitter.rs

```diff
@@ -210,6 +201,22 @@ impl SerialSubmitter {
             None => return Ok(()),
         };
 
+        if let Some(last_attempted_at) = msg.last_attempted_at {
+            if msg.num_retries > 1 {
+                let required_duration = Duration::from_secs(match msg.num_retries {
+                    0 | 1 => unreachable!(),
```

Comment:
I worry this policy will be an issue for our e2e tests, which use a recipient that will reject messages if the block number is odd (I.e. 50% of the time)

So maybe we keep the timeout low for like the first 16 attempts?

If you think about it, the overwhelming majority of the RPC requests are for messages that are never deliverable, and thus for a very high number of attempts

Reply:
I did test in CI and it was working as it was, but I went ahead and made it a bit more forgiving since the retry count is persisted, which means that on a relayer restart we are not going to end up going through all the retry backoffs again.